queries.sql

```diff
@@ -16,4 +16,31 @@ SELECT name FROM animals WHERE neutered;
 
 SELECT name FROM animals WHERE NOT name = 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+
```

Comment:
- [ ] It would be better if we write the query here in the queries.sql instead of writing it in the terminal as it is now
![Screenshot from 2022-07-22 16-13-57](https://user-images.githubusercontent.com/46388113/180481078-c65d04c1-7f6f-47a9-9973-e7e32d94a160.png)

Reply:
@addod19 I am not sure what you are implying or asking me to do here.